packages/teuchos/core/src/Teuchos_Utils.cpp

```diff
@@ -87,6 +87,13 @@ std::string Utils::toString(const int& x)
 	return std::string(s);
 }
 
+std::string Utils::toString(const long long& x)
```

Comment:
Teuchos already has a whole bunch of `toString` overloads, doesn't it?  How come Utils has its own `toString` functions?

Reply:
This isn't an issue for me that blocks the commit, but perhaps we should merge these `toString` implementations at some point.

Reply:
Yea, this ran through my mind as well when looking at these. I'm inclined to just call `std::tostring` now that we're C++11 and not have any custom ones. @bartlettroscoe  can comment.

Reply:
I don't remember the history of all of the toString() functions.  If C++11 functions do the job, I don't see why you would not use them I guess.

Reply:
@ibaned Be careful with `std::to_string` as it (weirdly) may give different results than `std::ostream& operator<<(std::ostream&, const T&)`:

http://en.cppreference.com/w/cpp/string/basic_string/to_string

In particular, it may change the number of significant digits.

Reply:
Yea, the standard says `to_string` behaves like `%f`, while our `toString` behaves like `%g`.

Reply:
@ibaned I was thinking a more modest change, namely to use the existing `Teuchos::toString` instead of `Teuchos::Utils::toString`.